CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Replaces manual thread management with cached thread pool
 - Files can now be moved to subfolders named by a custom format pattern, e.g., based on `entrytype`.
   The pattern can be specified in the settings like the filename pattern. [#1092](https://github.com/JabRef/jabref/issues/1092)
+  
```

Comment:
Ok, fine. I would merge this right now, but maybe we should give @bernhard-kleine at least time until the release tomorrow, before we merge and delete the branch. 

We should not forget to merge it before the release, though.